docs/deployment-guide.md

```diff
@@ -232,6 +232,10 @@ The automated deployment creates an instance of Sleeper either from your own con
 This also pre-populates certain properties for you, e.g. from your AWS configuration, and handles uploading the
 necessary deployment artifacts to AWS.
 
+We have planned to improve this by adding support for deploying a published version of Sleeper, and better support for
+declarative deployment with infrastructure as code. Please see the article
```

Comment:
The "better support for declarative deployment with infrastructure as code" bit here makes it sound like we don't currently support deploying this as infrastructure as code. The deployment is already infrastructure as code, so I think this should be clarified to say what else needs to be IaC.

Reply:
Done